gateway/handler_test.go

```diff
@@ -14,7 +14,7 @@ func setupTest(t *testing.T, method, target string, body io.Reader) *http.Respon
 	h, _ := getBasicHandler(t, &simulator.PlayBackMockConf{
 		BareDomain:      "example.com",
 		AccessKeyID:     "AKIAIO5FODNN7EXAMPLE",
-		AccessSecretKey: "MockAccessSecretKey",
+		SecretAccessKey: "MockSecretAccessKey",
```

Comment:
Since the request is already pre-signed for the test, this is failing it. Just bring it back to be "MockAccessSecretKey", so we don't need to replace the signature.